### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by nullstyle**
This PR rolls up a couple of fixes related to trades that didn't make it into yesterday's release.

### CODE REVIEW

File: src/github.com/stellar/horizon/test/t.go [hunk 1]
```diff
@@ -47,6 +51,22 @@ func (t *T) ScenarioWithoutHorizon(name string) *T {
 	return t
 }
 
+// UnmarshalPage populares dest with the records contained in the json-encoded
```
Comment (bartekn) [inline src/github.com/stellar/horizon/test/t.go]: `populates`

File: src/github.com/stellar/horizon/db2/core/order_book_summary_test.go [hunk 2]
```diff
@@ -85,17 +85,17 @@ func TestGetOrderBookSummary_Regress310(t *testing.T) {
 	// 0.1, 0.2, and 0.3 should appear first, when the
 	// query is correct.  In a failing scenario the 0.2
 	// transaction should not appear.
```
Comment (bartekn) [inline src/github.com/stellar/horizon/db2/core/order_book_summary_test.go]: Comment needs to be updated.

**Review by bartekn — COMMENTED** [review]
👍

File: src/github.com/stellar/horizon/db2/core/order_book_summary_test.go (hunk 2 above)
Comment (nullstyle) [reply src/github.com/stellar/horizon/db2/core/order_book_summary_test.go]: oh snap, thanks!  I'll fix it now.  I'm always bad at maintaining comments.

**Comment by jedmccaleb** [comment]
r+

**Pull request merged**